src/dyad/core/dyad_core.c

```diff
@@ -4,17 +4,17 @@
 #error "no config"
 #endif
 
+#include <dyad/common/dyad_flux_log.h>
 #include <dyad/core/dyad_core.h>
-
-#include <libgen.h>
-#include <unistd.h>
-#include <fcntl.h>
-
 #include <dyad/core/dyad_envs.h>
 #include <dyad/dtl/dyad_dtl_impl.h>
-#include <dyad/common/dyad_flux_log.h>
+#include <dyad/perf/dyad_perf.h>
 #include <dyad/utils/murmur3.h>
 #include <dyad/utils/utils.h>
+#include <libgen.h>
+#include <unistd.h>
+#include <fcntl.h>
```

Comment:
I like the ordering of headers you are exercising here. including from specific to generic c headers.